app/assets/javascripts/entries.js

```diff
@@ -7,12 +7,40 @@ function showOrHideLinkedEntry(){
   }
 }
 
+$(document).ready(function() {
+    // Zamiana przecinków na kropki w czasie rzeczywistym
+    $('.amount-input').on('input', function() {
+        var value = $(this).val().replace(/,/g, '.');
+        $(this).val(value);
+
+    $('.amount-input-one-percent').on('input', function() {
+        var value = $(this).val().replace(/,/g, '.');
+        $(this).val(value);
+        updateTotalSum('.amount-input-one-percent', '#total-sum-one-percent');
+    });
+
+    $('.amount-input-grants').on('input', function() {
+        var value = $(this).val().replace(/,/g, '.');
+        $(this).val(value);
+        var grantId = $(this).siblings('.grant_id').val();
+        if (grantId) {
+            updateTotalSum('.amount-input-grants.grant-' + grantId, '#total-sum-grant-' + grantId);
+        }
+    });
+        // Aktualizuj sumę po każdej zmianie
+        updateTotalSum('.amount-input', '#total-sum');
+    });
+
+    // Początkowa aktualizacja sumy
+    updateTotalSum('.amount-input', '#total-sum');
+});
```

Comment:
This function works OK but it might make using the app a bit awkward.
It's perfectly fine to replace commas with dots for summary fields only.

Please remove that section.

Reply:
```suggestion
```